deps.ts

```diff
@@ -1,40 +1,35 @@
 export {
   serve,
   ServerRequest
-} from "https://deno.land/std@v0.32.0/http/server.ts";
+} from "https://deno.land/std@v0.34.0/http/server.ts";
```

Comment:
```
const DENO_STD_URL = `https://deno.land/std@v0.34.0/`

export {
 serve,
 ServerRequest
} from `{DENO_STD_URL}http/server.ts`;
```

Just a thought.... might start to get cumbersome if this file gets bigger. Maybe this is something that we are already thinking about adding to the minimal imports feat... in that case, would be good to have a prop for this deno_std_url?

Reply:
That sounds good, but I'm pretty sure that would require dynamic imports since variables can't go into the URL. I could add in the `bump_versions` script again. It took a `from` and a `to` variable and would update the versions `from` #.#.# `to` #.#.#. I'll make an issue about it though.